examples/map/track_and_coalign_region.py

```diff
@@ -0,0 +1,69 @@
+"""
+=========================================
+Tracking and Co-aligning an Active Region
+=========================================
+
+This example demonstrates how to track an active region as it rotates across the solar disk
+and make cutouts around that active region at each time step to build an aligned datacube.
+"""
+
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+from astropy.visualization import ImageNormalize, SqrtStretch
+
+import sunpy.map
+from sunpy.coordinates import propagate_with_solar_surface
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+
+###############################################################################
+# First, let's download a series of images in time using `~sunpy.net.Fido`.
+# In this example, we will download a series of AIA 171 Å images observed over the course
+# of one day at a cadence of 1 image every 3 hours.
+
+query = Fido.search(a.Time('2018-05-29 18:00:00', '2018-05-31 06:00:00'),
```

Comment:
There seems to be a dark frame in this sequence, we might need to adjust the time by 15 minutes and see if that helps.

Reply:
I think my commit should now fix this. Apologies for my previous suggested date range including some bad frames.

Reply:
There was no way to know anyway.